devops-infra/.htaccess

```diff
@@ -1,5 +1,48 @@
 Header set Access-Control-Allow-Origin *
 Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
 Options +FollowSymLinks
-RewriteEngine on
-RewriteRule ^.*$ https://github.com/oeg-upm/devops-infra [R=303,L]
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .json
+# Rewrite engine setup
+RewriteEngine On
+
+#If no specific ontology is requested, return the HTML ontology catalogue 
+RewriteRule ^.*$ https://oeg-upm.github.io/devops-infra/index.html [R=302,L]
```

Comment:
If you have this rule first, it will superseed all subsequent rules (you are matching anything). You should have this one at the end